---
layout: vanilla
permalink: /data/
description: "Data for the Chromatin Cell Atlas of the Developing Fly Embryo."
modified: 2018-01-16
tags: [fly, data, atlas, analysis]
---
<!DOCTYPE html>
<html lang= "{{ page.lang | default: site.lang | default: "en" }}">

  {% include head.html %}

  <!-- See here for how to use the TOC control:
  https://afeld.github.io/bootstrap-toc/ -->
  <body  data-spy= "scroll" data-target= "#toc">
    {% include header.html %}


    <div class= "container">
      <div class= "row">
        <!-- sidebar, which will move to the top on a small screen -->
        <div class= "col-sm-4 sidenav">
          <nav id= "toc" data-spy= "affix" data-toggle= "toc"></nav>
        </div>
        <!-- main content area -->
        <div class= "col-sm-8">

   <h2>Release Updates</h2>
  <ul>
    <li><code>06/09/2019</code> adds <code>summitmatrix.sparse.binary.rds</code>, <code>tsnecoords.txt.gz</code>, and <code>densitypeakclusters.txt.gz</code> files for all three timepoints.</li>
    <li><code>03/14/2018</code>: Initial release.</li>
  </ul>
  <div class="hugespacer"></div>
          
          
    <h2>Published Data</h2>
    <p>On this page, we make some of the raw data files available for conducting your own analyses.</p>

    <h2>Tutorial Data</h2>
    <p>Here are links to all of the data used in the tutorial (code is presented for downloading and processing these files through the course of the tutorial too).

            <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/irlba_1.0.3.tar.gz" download>irlba_1.0.3.tar.gz</a></th>
                  <th>18-Nov-2017 17:20</th>
                  <th>177K</th>
                  <th>Source code for <code>irlba</code> version 1.0.3, the version used for latent semantic indexing in Cusanovich, Reddington, Garfield <i>et al.</i></th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/monocle_2.5.3.tar.gz" download>monocle_2.3.5.tar.gz</a></th>
                  <th>18-Nov-2017 17:20</th>
                  <th>16M</th>
                  <th>Source code for <code>Monocle</code> version 2.3.5, the version used for pseudotemporal ordering in Cusanovich, Reddington, Garfield <i>et al.</i></th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/DDRTree_0.1.4.tar.gz" download>DDRTree_0.1.4.tar.gz</a></th>
                  <th>16-Jan-2018 11:23</th>
                  <th>12K</th>
                  <th>Source code for <code>DDRTree</code> version 0.1.4, the version used for pseudotemporal ordering in Cusanovich, Reddington, Garfield <i>et al.</i></th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/monocle_patch.R" download>monocle_patch.R</a></th>
                  <th>18-Nov-2017 17:20</th>
                  <th>6.6K</th>
                  <th>A small update to Monocle that causes the <code>differentialGeneTest</code> to return beta values.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.2kbmatrix.sparse.binary.rds" download>6to8.2kbmatrix.sparse.binary.rds</a></th>
                  <th>18-Nov-2017 17:20</th>
                  <th>113M</th>
                  <th>A binary matrix recording whether insertions were observed in 2kb windows throughout the genome in all 6-8 hour cells. This file is provided as a sparse matrix in <code>.rds</code> format so that it can be easily loaded into <code>R</code>.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.summitmatrix.sparse.binary.rds" download>6to8.summitmatrix.sparse.binary.rds</a></th>
                  <th>18-Nov-2017 17:19</th>
                  <th>69M</th>
                  <th>A binary matrix recording whether insertions were observed in a master list of potential regulatory elements in all 6-8 hour cells. This file is provided as a sparse matrix in <code>.rds</code> format so that it can be easily loaded into <code>R</code>.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.xycalls.txt" download>6to8.xycalls.txt</a></th>
                  <th>18-Nov-2017 17:19</th>
                  <th>300K</th>
                  <th>A file assigning a sex to each cell from the 6-8 hour time point on the basis of reads mapping to the sex chromosomes. A <code>1</code> indicates male and a <code>2</code> indicates female.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.read.report.txt" download>6to8.read.report.txt</a></th>
                  <th>18-Nov-2017 17:19</th>
                  <th>429K</th>
                  <th>A file reporting the total number of unique reads recovered from each cell. This was used in the logistic regression analysis to control for cells having varying complexity.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/2to4_files.tar.gz" download>2to4_files.tar.gz</a></th>
                  <th>18-Nov-2017 17:19</th>
                  <th>44M</th>
                  <th>A tarball of all the files required for pseudotemporal ordering of cells from the 2-4 hour time point.</th>
                </tr>
                </tbody>
              </table>
          
              <h2>Accessibility Matrices</h2>
    <p>Here we make peak of accessibility by cell matrices for all three time points. Each matrix is a sparse matrix file in <code>.rds</code> format. Requires the <code>Matrix</code> package in R to open.</p>
          
          <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/2to4.summitmatrix.sparse.binary.rds" download>2to4.summitmatrix.sparse.binary.rds</a></th>
                  <th>8-Jun-2019 07:05</th>
                  <th>59M</th>
                  <th>Accessiblity matrix for the 2-4hr timepoint.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.2kbmatrix.sparse.binary.rds" download>6to8.2kbmatrix.sparse.binary.rds</a></th>
                  <th>18-Nov-2017 17:19</th>
                  <th>113M</th>
                  <th>Accessiblity matrix for the 6-8hr timepoint. The same file is also available in the Tutorial section and is just reproduced here for consistency.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/10to12.summitmatrix.sparse.binary.rds" download>10to12.summitmatrix.sparse.binary.rds</a></th>
                  <th>9-Jun-2019 14:37</th>
                  <th>48M</th>
                  <th>Accessiblity matrix for the 10-12hr timepoint.</th>
                </tr>
                </tbody>
              </table>

          
                        <h2>t-SNE Coordinates</h2>
    <p>Here we make t-SNE coordinates for each timepoint available. See the manuscript for how t-SNE coordinates were calculated. Each file is a gzipped .tsv and includes 3 columns: Cell ID column, x-axis coordinate column, y-axis coordinate column.</p>
          
          <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/2to4.tsnecoords.txt.gz" download>2to4.tsnecoords.txt.gz</a></th>
                  <th>9-Jun-2019 15:12</th>
                  <th>165K</th>
                  <th>t-SNE coordinates for the 2-4hr timepoint.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.tsnecoords.txt.gz" download>6to8.tsnecoords.txt.gz</a></th>
                  <th>9-Jun-2019 15:12</th>
                  <th>162K</th>
                  <th>t-SNE coordinates for the 6-8hr timepoint.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/10to12.tsnecoords.txt.gz" download>10to12.tsnecoords.txt.gz</a></th>
                  <th>9-Jun-2019 15:12</th>
                  <th>149K</th>
                  <th>t-SNE coordinates for the 10-12hr timepoint.</th>
                </tr>
                </tbody>
              </table>

                                <h2>Cluster Assignments</h2>
    <p>Here we make the density peak-based cluster assignments (derived from the t-SNE coordinates) for each timepoint available. See the manuscript for how t-SNE coordinates were calculated and how density peak clusters were identified. Each file is a gzipped .tsv and includes 2 columns: Cell ID column, cluster assignment.</p>
          
          <table class = "table">
              <thead>
                <tr>
                  <th>Name</th>
                  <th>Last modified</th>
                  <th>Size</th>
                  <th width = "50%">Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/2to4.densitypeakclusters.txt.gz" download>2to4.densitypeakclusters.txt.gz</a></th>
                  <th>8-Jun-2019 05:42</th>
                  <th>34K</th>
                  <th>Cluster assignments for the 2-4hr timepoint.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/6to8.densitypeakclusters.txt.gz" download>6to8.densitypeakclusters.txt.gz</a></th>
                  <th>8-Jun-2019 05:42</th>
                  <th>34K</th>
                  <th>Cluster assignments for the 6-8hr timepoint.</th>
                </tr>
                <tr>
                  <th><a href = "http://krishna.gs.washington.edu/content/members/cusanovich/fly_embryogenesis/updated_data/vignette/10to12.densitypeakclusters.txt.gz" download>10to12.densitypeakclusters.txt.gz</a></th>
                  <th>8-Jun-2019 05:41</th>
                  <th>31K</th>
                  <th>Cluster assignments for the 10-12hr timepoint.</th>
                </tr>
                </tbody>
              </table>

          

      </div>
    </div>
</div> <!-- /container -->

{% include footer.html %}

</body>

</html>
